### ISSUE TITLE
Permissions for running applescripts

### ISSUE THREAD

**Issue by ivan-unfolds** [martakostova/pomodoro#41]
@martakostova I was wondering if you can help!

I am trying to add the following applescript so that my computer locks itself at the end of pomodoro period. This is so I can make sure that I take my pomodoro break as I often miss the alarm and next thing I know 3 hours have gone by without a break.

```
activate application "SystemUIServer"
tell application "System Events"
    tell process "SystemUIServer" to keystroke "q" using {command down, control down}
end tell
```

However I am getting an error when trying to do this. After doing some research I think the script is failing because the app does not have the right permissions. I am not sure but I think the problem is that it is not in the `Automation` tab of the `Security and Privacy` settings.

<img width="780" alt="Screenshot 2019-05-07 at 19 57 23" src="https://user-images.githubusercontent.com/12462448/57325666-72b74f00-7102-11e9-89d6-f4c043eef429.png">

Do you know if there's anything that can be done?

**Comment by martakostova** [comment]
Yes, this is because of Sandboxing (which is requirement for the App Store). Here you can find instructions how to disable it https://github.com/martakostova/pomodoro/issues/19

**Comment by ivan-unfolds** [comment]
Thanks for speedy response @martakostova 🙂 

#19 didn't work unfortunately, when I changed `com.apple.security.app-sandbox` boolean, the app wouldn't open. However I found your other suggestions of building the app locally. This didn't work initially, however I added [NSAppleEventsUsageDescription](https://developer.apple.com/documentation/bundleresources/information_property_list/nsappleeventsusagedescription) to the `Pomodoro-info.plist` and it worked. Hopefully that is of help for anyone with same issue!

**Comment by martakostova** [comment]
@ivanmauricio this was very useful. Thanks! Will add it in the next version.
Cheers!

**Comment by ivan-unfolds** [comment]
No worries, glad I could be of help 😊 

As an aside I have been using that script for the last few days and it's worked really well at helping me stick to my breaks. I know you take feature requests, so I think an option to lock your computer at break ends is a good one. Not sure how hard that would be to implement though!

**Comment by JosepFrancesc** [comment]
Hi @martakostova ,
I have the same problem with Catalina. Can you help me please?
